### DISCUSSION TITLE
相同参数，语速相差非常大

### DISCUSSION THREAD

**Discussion by GwsinTracy** [fishaudio/Bert-VITS2#381] [General]
相同的length_scale参数，不同的启动后，生成的音频语速相差非常大。看了下这个是StochasticDurationPredictor模型生成的logw -> w -> y_lengths -> y_mask导致每次合成的长度不一样。
同一次启动的情况下，y_lengths相差不会很大，但是不同的启动下，y_lengths相差会非常大。
请问这个有人遇到过吗？可能会是什么原因导致的？

**Comment by GwsinTracy** [comment]
找到问题所在了，忽略。

**Reply by CoCoColaZ** [reply]
是什么原因呢 我也有这种情况
